### DISCUSSION TITLE
Does the LRN operator only support the calculation of 4-dimensional tensors?

### DISCUSSION THREAD

**Discussion by git-ql** [apache/incubator-mxnet#14827] [Q&A]
## Does the LRN operator only support the calculation of 4-dimensional tensors?

When I use LRN operator with an input data whose shape is（6，4，9）, an  error occurred and log info is as follows:

![call stack](https://user-images.githubusercontent.com/50092277/56879384-12ce0200-6a8b-11e9-82bc-cc4c341a9052.jpg)

## Environment info 

```
mxnet 1.3.0
CUDA 9.1
UBUNTU

Package used (Python/R/Scala/Julia):
python 2.7

```

[a comment by an automated account is not shown]

**Comment by lanking520** [comment]
@git-ql Have you tried with `v.1.4.0`, does this problem persist?
@anirudh2290 @stu1130 could you please take a look?

**Comment by stu1130** [comment]
Could we have some reproducible script?
I checked the LRU implementation, it looks like we do only support 4 dimensions

**Comment by git-ql** [comment]
> @git-ql Have you tried with v.1.4.0, does this problem persist?
> @anirudh2290 @stu1130 could you please take a look?

After reviewing the code of  LRN operator, we find that the implement in v.1.4.0 is similar to that in v.1.3.0 and also only support 4-dimentiaonal tensors.

**Comment by git-ql** [comment]
> Could we have some reproducible script?
> I checked the LRU implementation, it looks like we do only support 4 dimensions

My script is as follows:
```
import mxnet as mxnet
import numpy as np

data_shape = tuple(np.random.randint(2, 10, size=3))

local_tmp = int(np.random.uniform(1, data_shape[1]))
local_size = local_tmp if local_tmp%2 else local_tmp-1

alpha = round(np.random.uniform(0.0001, 0.0003), 4)
beta = round(np.random.uniform(0.5, 0.8), 1)
k = np.random.randint(1, 5)

data = mx.sym.Variable("data")
data_array = mx.nd.array(np.random.uniform(1, 100, data_shape))

lrn = mx.sym.LRN(data=data, alpha=alpha, beta=beta, knorm=k, nsize=local_size)

exe_lrn = lrn.simple_bind(ctx=mx.cpu(0), data=data_shape, grad_req="null")
exe_lrn.arg_dict["data"][:] = data_array
exe_lrn.forward()

print("lrn_output: {}".format(exe_lrn.outputs[0].asnumpy()))
```
